src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/BulkPrintService.java

```diff
@@ -31,8 +31,8 @@
     private static final String DOCUMENT_URL = "document_binary_url";
     private static final String VALUE = "value";
 
-    @Value("${feature.approved-consent-order-notification-letter}")
-    private boolean approvedConsentOrderNotificationLetterFeature;
+    @Autowired
```

Comment:
`Autowired` is deprecated - where possible use `private final` for services and Lombok's `@RequiredArgsContructor` ? or is there a reason to leave it package proctected?